modules/objc/examples/code_snippets/SampleCodeTest.m

```diff
@@ -1804,7 +1804,48 @@ - (void) dontTestQuerySyntaxJson {
 
 // end::query-syntax-pagination[]
 
+- (void) docsonly_QuerySyntaxN1QL() {
+    /* Documentation Only query-syntax-n1ql
+        Declared elsewhere: CBLDatabase* argDb
+    */
 
+    // tag::query-syntax-n1ql[]
+    NSError *error;
+
+    CBLDatabase *db == argDB;
+
+    NSString *n1qlstr = "SELECT * FROM _ WHERE type = \"hotel\""; // <.>
```

Comment:
not sure, should we promote users to write N1QL query using `_` or with a valid database name?

Reply:
As far as I am aware Jay _ is fine to use; may change when collections and scopes are supported